TypeInjections/TypeInjections.Test/DummyMain.fs

```diff
@@ -0,0 +1,7 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT
+
+module TypeInjections.Test.DummyMain
```

Comment:
Can we name `DummyMain` something more informative?

Reply:
Changed name of DummyMain to Tester